scripts/start.sh

```diff
@@ -98,6 +115,22 @@ function main() {
       setup_sqlflow_server &
       setup_sqlflow_notebook
       ;;
+    repl)
+      shift
+      . ~/.sqlflow_env
+      if [[ $SQLFLOW_DATASOURCE == "" ]]; then
+        echo "No data source specified. Starting builtin MySQL"
+        start_mysql
+        sleep_until_mysql_is_ready
+        SQLFLOW_DATASOURCE="mysql://root:root@tcp(${SQLFLOW_MYSQL_HOST}:${SQLFLOW_MYSQL_PORT})/?maxAllowedPacket=0"
+	  fi
+	  export SQLFLOW_DATASOURCE
+      if [[ ! -d /var/lib/mysql/iris && $SQLFLOW_DATASOURCE == mysql://* ]]; then
```

Comment:
Does this mean that the command-line tool works only with MySQL?  Could it work with MaxCompute?

Reply:
> Does this mean that the command-line tool works only with MySQL? Could it work with MaxCompute?

If the user doesn't create a config file after downloads. (Type `./sqlflow` and enter). The client will connect to the `mysqld` nested in the docker container. If the user has specified a connection string, the client will connect to the specified database server. The databases supported at the moment are MySQL, Hive, and MaxCompute.